### PR TITLE
Template update for nf-core/tools v2.14.1

### PR DESCRIPTION
**Pull request by WackerO**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/qbic-pipelines/rnadeseq/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/workflows/release-announcements.yml
Comment (d4straub) [inline .github/workflows/release-announcements.yml:1]: Is that functional? Probably we dont want to have that?

File: assets/samplesheet.csv
Comment (d4straub) [inline assets/samplesheet.csv:1]: This isnt used, is it?

File: assets/schema_input.json
Comment (d4straub) [inline assets/schema_input.json:1]: Also not used, right?

File: subworkflows/local/utils_nfcore_rnadeseq_pipeline/main.nf [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+        }
+        .groupTuple()
+        .map {
+            validateInputSamplesheet(it)
+        }
+        .map {
+            meta, fastqs ->
+                return [ meta, fastqs.flatten() ]
+        }
+        .set { ch_samplesheet }
```
Comment (d4straub) [inline subworkflows/local/utils_nfcore_rnadeseq_pipeline/main.nf:101]: Is that used at all?

File: subworkflows/local/utils_nfcore_rnadeseq_pipeline/main.nf [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+    def (metas, fastqs) = input[1..2]
+
+    // Check that multiple runs of the same sample are of the same datatype i.e. single-end / paired-end
+    def endedness_ok = metas.collect{ it.single_end }.unique().size == 1
+    if (!endedness_ok) {
+        error("Please check input samplesheet -> Multiple runs of a sample must be of the same datatype i.e. single-end or paired-end: ${metas[0].id}")
+    }
+
+    return [ metas[0], fastqs ]
+}
```
Comment (d4straub) [inline subworkflows/local/utils_nfcore_rnadeseq_pipeline/main.nf:174]: not used?

**Review by d4straub — APPROVED** [review]
Havent dont the template update myself yet, so might miss something ;)

File: .github/workflows/release-announcements.yml
Comment (WackerO) [reply .github/workflows/release-announcements.yml:1]: The tokens are not defined AFAIK, so I think it should not work. But just to be sure I'll delete the file

File: assets/samplesheet.csv
Comment (WackerO) [reply assets/samplesheet.csv:1]: nope!

File: assets/schema_input.json
Comment (WackerO) [reply assets/schema_input.json:1]: will also delete

File: subworkflows/local/utils_nfcore_rnadeseq_pipeline/main.nf (hunk 1 above)
Comment (WackerO) [reply subworkflows/local/utils_nfcore_rnadeseq_pipeline/main.nf:101]: no, will remove the parts of the file that are not used (which is most of them)

File: subworkflows/local/utils_nfcore_rnadeseq_pipeline/main.nf (hunk 2 above)
Comment (WackerO) [reply subworkflows/local/utils_nfcore_rnadeseq_pipeline/main.nf:174]: see above

**Comment by WackerO** [comment]
Thanks Daniel!

**Pull request merged**